Travis CI build failing

**Checklist**
- [x] Bug Report - The issue you are reporting might be there already. Please search on the [issues tracker](https://github.com/openfoodfacts/openfoodfacts-androidapp/issues) before creating or reporting an issue.
- [x] Before reporting, please ensure (if you can) that you have tested it with the latest (dev) version of the software.
- [x] Don't forget to tag your issue, it makes it easier to discover.

**Summary:** 
```
FAILURE: Build failed with an exception.
What went wrong:
Could not resolve all files for configuration ':app:obfDebugCompileClasspath'.
Could not find support-core-ui.aar (com.android.support:support-core-ui:27.1.1).
```

--- a/build.gradle
+++ b/build.gradle
@@ -19,10 +19,10 @@ buildscript {
 
 allprojects {
     repositories {
+        google()
         mavenCentral()
         jcenter()
         maven { url "https://jitpack.io" }
-        google()
     }
 }
 

